src/libostree/libostree.sym

```diff
@@ -301,6 +301,7 @@ global:
         ostree_sysroot_write_origin_file;
         ostree_validate_checksum_string;
         ostree_validate_rev;
+        ostree_validate_remote_name;
```

Comment:
This is the wrong place; you're not the first to do this though.  I submitted https://github.com/ostreedev/ostree/pull/931